### ISSUE TITLE
To, CC and BCC fields seem to have issues with screenreaders

### ISSUE THREAD

**Issue by domasofan** [freescout-helpdesk/freescout#5382]
Hi all,

It seems that the To, CC and BCC fields seem to have issues with screenreaders in the new conversation form.
Search and autosuggestion/completion seems not to work.
At least it seems not to work in the demo, which seems to run still on version 1.8.207.
Would be cool if someone could have a look into this and maybe also update the demo to the recent version.

We plan to use FreeScout in our office and i am one of the admins who should be fully use it with a screenreader.
I am using NVDA (https://www.nvaccess.org/) under Windows.
Though it should be also possible to replicate the issue with Orca on Linux, VoiceOver on Mac and others.

Greetings and thanks,
Simon

**Comment by freescout-help** [comment]
We've forwarded your issue to this discussion: https://github.com/freescout-help-desk/freescout/issues/1150
